edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/cluster/KindKubernetesClientProvider.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Cluster
+{
+    using System.Threading.Tasks;
+    using CliWrap;
+    using k8s;
+
+    public class KindKubernetesClientProvider : IKubernetesClientProvider
+    {
+        readonly string name;
+
+        public KindKubernetesClientProvider(string name)
+        {
+            this.name = name;
```

Comment:
is it required?

Reply:
Yeah. In order to run some tests in parrallel you may want to create separate cluster with different names.

Reply:
This class is obsolete and to be deleted.